### ISSUE TITLE
AtlServer build error

### ISSUE THREAD

**Issue by forderud** [jogerh/com_samples#24]
I'm experiencing the following problem when attempting to build AtlServer with VS2019:
```
1>------ Build started: Project: AtlServer, Configuration: Debug x64 ------
1>AtlServer.rc(13): fatal error RC1015: cannot open include file 'afxres.h'.
1>
1>Done building project "AtlServer.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 1 up-to-date, 0 skipped ==========
```

**Comment by forderud** [comment]
@jogerh Any clue about this one?

**Comment by jogerh** [comment]
Please have a look at #25. Will this resolve the issue?

**Comment by forderud** [comment]
The problem was fixed by PR #25 .
